gamestonk_terminal/stocks/behavioural_analysis/ba_controller.py

```diff
@@ -127,9 +131,11 @@ def print_help(self):
     metrics       core social sentiment metrics for this stock
     social        social media figures for stock popularity
     historical    plot the past week of data for a selected metric{res}
+    his           plot historical RHI and AHI data by hour
```

Comment:
So these old SI functions can be removed from this string.  And the {res} should be moved to the end of the his function.

Also can we rename it `hist`

Reply:
what's our intention with the {res} and {dim} styling? With {dim} the text is greyed out when the ticker is not yet loaded.

Reply:
So the {res} was just to reset the style so everything after wasn't dim (and the dim indicates you need a ticker to use the function as you say).  We are actually completely changing the styling in PR #1161 so feel free to ignore this comment for now